### ISSUE TITLE
[Feature Request] Re-add (and possibly enhance) "Mine" avatars list in the search tab.

### ISSUE THREAD

**Issue by Janni9009** [Natsumi-sama/VRCX#19]
While the vanilla API does not allow for search and the public and recent endpoints are long dead, the Mine tab was actually useful for displaying a nice, large, and page-based view of your avatars sorted by most recent, which was very useful for quickly assigning images to them, changing upload status, etc.. The on-profile avatars tab in comparison, while it works fine, feels rather small and squished together.

Perhaps some improvements to it would be nice to see as well, like showing the avatar preview in a full rectangle, showing description, version and upload status on it, and having a button for features like copying the avatar id, uploading an image, and changing the name. sort of like a compact popout list with the most important features on it if that's not too difficult to implement.

**Comment by Murkur** [comment]
I have to agree with this especially with people with high amounts of avatars the small box provided is really kind of unusable I only have around 102 avatars and even then its cumbersome and I know way more people with over 400.

Before After Comparison

Before:
![VRCX_7OYwCXzgl5](https://user-images.githubusercontent.com/20288698/136297071-91dddb16-f5ac-4788-b0bd-ba651296a704.png)


After:
![VRCX_8SLUwO1132](https://user-images.githubusercontent.com/20288698/136297085-85d7b04e-cadd-453e-839c-4aae38ce362c.png)

**Comment by Natsumi-sama** [comment]
Hmmm what would be preferred, make the window bigger or just add the tab back?

**Comment by Murkur** [comment]
Tbh the old tab is kind of outdated it would be nice to just duplicate the current avatars tab if anything and to the same location so we also can see Publication status of avatars what you think @Janni9009 ?

**Comment by Janni9009** [comment]
Yeah that sounds like a good idea @Murkur, but adding to that it should be a list as well rather than a grid so names don't get cut off.

**Comment by Murkur** [comment]
agreed keep it as a list but update the actual containers to have publication status like the current one 👍
